Tests/Unit/EmogrifierTest.php

```diff
@@ -948,6 +948,41 @@ public function emogrifyAppliesCssFromStyleNodes()
     /**
      * @test
      */
+    public function emogrifyDoesNotApplyCssFromStyleBlocksWhenDisabled()
+    {
+        $styleAttributeValue = 'color: #ccc;';
+        $html = $this->html5DocumentType . self::LF .
+            '<html><style type="text/css">html {' . $styleAttributeValue . '}</style></html>';
+        $this->subject->setHtml($html);
+        $this->subject->disableStyleBlocksParsing();
+
+        $this->assertNotContains(
+            '<html style="' . $styleAttributeValue . '">',
+            $this->subject->emogrify()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function emogrifyDoesNotApplyCssFromInlineStylesWhenDisabled()
+    {
+        $html = $this->html5DocumentType . self::LF .
+            '<html><head><style>#topWrap p {padding-bottom: 1px;PADDING-TOP: 0;}</style></head>'
+            . '<body><div id="topWrap"><p style="text-align: center;">some content</p></div></body></html>';
+        $expected = '<p style="padding-bottom: 1px; padding-top: 0;">';
```

Comment:
Currently, this test also tests that style rules are overwritten. Could you please simplify it so that it's down to the bare bones?